weblate/static/editor/browse.js

```diff
@@ -0,0 +1,28 @@
+(() => {
+  function ProjectStringsBrowser() {
+    Mousetrap.bindGlobal("right", (e) => {
+      const nextButton = $("#button-next");
+      const nextLocation = nextButton.attr("href");
+      if (nextButton.length && !nextButton.hasClass("disabled")) {
+        if (nextLocation != undefined) {
+          window.location.href = nextLocation;
+        }
+      }
+      return false;
+    });
+    Mousetrap.bindGlobal("left", (e) => {
+      const prevButton = $("#button-prev");
+      const prevLocation = prevButton.attr("href");
+      if (prevButton.length && !prevButton.hasClass("disabled")) {
+        if (prevLocation != undefined) {
```

Comment:
```suggestion
        if (prevLocation !== undefined) {
```

See https://github.com/WeblateOrg/weblate/actions/runs/10006505825/job/27659356277?pr=12093#step:7:106